.phan/config.php

```diff
@@ -31,6 +31,10 @@
 		"PhanTypeMismatchProperty",
 		"PhanNonClassMethodCall",
 		"PhanTypeArraySuspicious",
+        "PhanTypeExpectedObjectPropAccess",
+        "PhanTypeSuspiciousStringExpression",
+        "PhanContinueTargetingSwitch",
+        "PhanTypeVoidAssignment"
```

Comment:
Weren't PhanTypeExpectedObjectPropAccess and PhanTypeVoidAssignment already fixed?

Reply:
Oops yes, but I saw new errors being generated. Not sure if that's due to a bad merge into major or because Phan got better at detecting errors. I'll remove these rules again and also issue fixes for the errors flagged.

Reply:
Could also be due to a bug fix in phan where it's catching things that it didn't previously

Reply:
If it's catching new things we can still upgrade it and then fix them in separate PRs

Reply:
Nah they're two quick one-liners. May as well do it here.